packages/pkcs12/src/__tests__/pkcs12-test.ts

```diff
@@ -0,0 +1,101 @@
+  password: 'password',
+  alias: 'test-alias',
+  md5Fingerprint: 'CCD9D0FD20A862B1E67EA838C99DD0C5',
+  sha1Fingerprint: '2EF9743A1BF12189AF5503787B54BFD3254B46A8',
+  sha256Fingerprint: 'EECDD35E90BB272D52EC261B4063C5D586980795452340525F9C8CC56DB53E2B',
+};
+describe('PKCS12 Tests', () => {
+  it('Conventional P12 Cert Fingerprints', async () => {
```

Comment:
Write test descriptions to fit in the sentence, "It ...":
```suggestion
  it('computes fingerprints of certificates in conventional p12 files', async () => {
```